app/dashboard/templatetags/math_filters.py

```diff
@@ -0,0 +1,14 @@
+import re
```

Comment:
what is this file for?

Reply:
It is no longer relevant. Will delete @danlipert